votrfront/js/coursesStats.js

```diff
@@ -63,7 +63,7 @@ export function currentAcademicYear() {
   var year = date.getFullYear();
   var month = date.getMonth() + 1;
```

Comment:
Hmm, vlastne moment. Nevsimol som si ze uz robime `+ 1`. Takze aj ked getMonth() vracia 0..11, month je 1..12. Zda sa ze tato funkcia uz aj doteraz ratala august do noveho akademickeho roku.

Reply:
ano, vratila som to na <8, teraz to vracia novy akademicky rok od augusta, predpokladam, ze tak to teda chceme